src/akamai/js/akamaiHD_flash.js

```diff
@@ -201,8 +204,31 @@ require("../../../html5-common/js/utils/constants.js");
      * @returns {boolean} True or false indicating success
      */
     this.setVideoUrl = function(url, encoding) {
-      /* Should be set to false if stream URL changes */
-      loaded = false;
+      var urlChanged = false;
+      newController=this.controller;
```

Comment:
it makes sense to make a variable that you can pass to callbacks but why is this getting set in setVideoUrl api?  Doesn't seem related in any way